woof-code/packages-templates/fonts-noto_FIXUPHACK

```diff
@@ -13,7 +13,7 @@ cat << EOF > etc/fonts/conf.d/01-noto.conf
 <?xml version="1.0"?>
 <!DOCTYPE fontconfig SYSTEM "fonts.dtd">
 <fontconfig>
-  <its:rules xmlns:its="http://www.w3.org/2005/11/its" version="1.0">
+  <its:rules xmlns:its="https://www.w3.org/2005/11/its" version="1.0">
```

Comment:
Change like this are breaking changes

Reply:
Done -- all http links are left intact now.